tests/stdlib/trandom.nim

```diff
@@ -57,3 +57,15 @@ randomize(223)
 
 for i in 0 .. 10:
   main()
+
+
+block:
```

Comment:
We should really remove
```
discard """
  #action: compile
"""
```
before anything else; 

in fact if I simply comment out
```
  randomize(223)
  for i in 0 .. 10:
    main()
```
in this test, `nim r tests/stdlib/trandom.nim` fails with `RangeDefect`

so I recommend following:
change this:
```nim
discard """
  action: compile
"""
```
to this:
```nim
discard """
  targets: "c cpp js"
  joinable: false
"""
```
(joinable: false so that the global random state doesn't affect other megatest joinable tests)

* then disable calling `main()` with a comment: `# TODO: fix properly so it can be run`

* then make sure the tests you've added work (right now they fail)